### PR TITLE
ACPENG-1385: Update kubecertmanager helper.go getRoute53HostedDomains() function

### PR DESCRIPTION
**Pull request by chej-hod**
getRoute53HostedDomains() has been updated to retrieve all hosted zone above the 100 api limit (pagination-ish)

### CODE REVIEW

**Review by jremes-ho — APPROVED** [review]
Provisionally LGTM. ✅

**Review by kmahmood2021 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.13
+FROM alpine:3.19
 MAINTAINER Rohith Jayawardene <gambol99@gmail.com>
```
Comment (RobJohnson-HO) [inline Dockerfile:2]: Small thing but shall we change this to ACP platform team email as maintainer?

**Review by RobJohnsonHO — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (chej-hod) [reply Dockerfile:2]: That's a good point, I'll get the maintainer updated

**Review by RobJohnsonHO — APPROVED** [review]
LGTM

**Review by vjremotegithub — APPROVED** [review]
LGTM

**Review by kmahmood2021 — APPROVED** [review]

**Pull request merged**